src/main/java/org/spongepowered/api/server/Server.java

```diff
@@ -242,4 +245,15 @@
      * @param timeout The player idle timeout
      */
     void setPlayerIdleTimeout(int timeout);
+
+    /**
+     * Gets whether this server is dedicated to being a global server, or
+     * whether this server is local to a game client where a {@link Client}
+     * instance may be available. The primary difference will be the types
+     * of {@link Player players} there may exist in the server, whether
+     * they are {@link ServerPlayer server players} or {@link RemoteClientPlayer remote players}.
+     *
+     * @return True if this is a dedicated server without a game client
+     */
+    boolean isDedicatedServer();
```

Comment:
The main difference is functionalities like rcon, etc. which would be better represented by a difference in inheritance.

Reply:
Still better to have both inheritance and a simple method to find out if the server is local or not. Since everything API wise wraps around the idea of "plugins are running on the server", we still have to expose the fact that the server could be remotely controlled, i.e. in the client connecting to a game server, there's still an integrated server.

Reply:
sure...

Reply:
isDedicatedServer, unlike isLocal on Player, makes sense per Gabe's rebuttal.